README.rst

```diff
@@ -107,10 +96,17 @@ For production
 To run tests
 ------------
 
+Tests won't run from `/vagrant` inside the vagrant system due to shortcomings
+of the way Vagrant makes the host system's files available there.  It's probably
+simplest to just do development directly on any Ubuntu 14 system.
```

Comment:
It may be worth mentioning that [Travis testing](https://travis-ci.org/PyCon/pycon) is done for pull requests - if worse comes to worse for developers who are not on recent Ubuntu systems.

Speaking of Travis testing - it would be nice to include the badge at the top of the repo.